packages/api-derive/src/util/memo.ts

```diff
@@ -0,0 +1,13 @@
+
+// Copyright 2017-2019 @polkadot/api-derive authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import memoizee from 'memoizee';
+
+export function memo <T extends Function> (fn: T): T {
```

Comment:
Can put this one too in rpc-core, it's the same code as https://github.com/polkadot-js/api/blob/master/packages/rpc-core/src/index.ts#L298-L306

Reply:
Almost the same... however length is not added.